core/account-id/src/lib.rs

```diff
@@ -297,28 +297,38 @@ impl AccountId {
     /// ```
     /// use near_account_id::AccountId;
     ///
-    /// let alice = AccountId::new_unvalidated("alice.near".to_string());
+    /// let alice = AccountId::new_unvalidated("alice.near");
     /// assert!(AccountId::validate(alice.as_ref()).is_ok());
     ///
-    /// let ƒelicia = AccountId::new_unvalidated("ƒelicia.near".to_string());
+    /// let ƒelicia = AccountId::new_unvalidated("ƒelicia.near");
     /// assert!(AccountId::validate(ƒelicia.as_ref()).is_err());
     /// ```
+    #[doc(hidden)]
     #[cfg(feature = "internal_unstable")]
     #[deprecated(since = "#4440", note = "AccountId construction without validation is illegal")]
-    pub fn new_unvalidated(account_id: String) -> Self {
-        Self(account_id.into())
+    pub fn new_unvalidated(account_id: impl Into<String>) -> Self {
+        Self(account_id.into().into())
     }
+}
 
-    pub fn is_system(&self) -> bool {
-        self.as_ref() == "system"
+#[doc(hidden)]
+#[allow(deprecated)]
+#[cfg(feature = "static_accounts")]
+impl AccountId {
+    pub fn near_account() -> Self {
+        Self::new_unvalidated("near")
```

Comment:
I wonder why do we use the 'invalidated' flow, rather than doing the 'normal' one ("near".parse()..)

as all these 4 names should pass the validation, right?

Reply:
Yeah, they will. But since we know these names to be valid, we can skip the unnecessary checks.